client/my-sites/plan-features/index.jsx

```diff
@@ -105,6 +111,11 @@ export default connect( ( state, ownProps ) => {
 		features: getPlanFeaturesObject( ownProps.plan ),
 		rawPrice: getPlanRawPrice( state, planProductId, ! isMonthly( ownProps.plan ) ),
 		planConstantObj: plansList[ ownProps.plan ],
+		available : canUpgradeToPlan( ownProps.plan ),
+		onUpgradeClick: () => {
```

Comment:
I started working on bringing the redesigned plans to NUX and stumbled upon a problem. In old/current plans, `PlanActions` provide `cartItem` to the `onUpgradeClick` method: https://github.com/Automattic/wp-calypso/blob/c1e4bb744e9266fdf066121c033972f3742e3295/client/components/plans/plan-actions/index.jsx#L167-L167. It is needed for `SignupActions`: https://github.com/Automattic/wp-calypso/blob/9cb2557484fc059ab3af01be5de03e913770abf1/client/signup/steps/plans/index.jsx#L66-L66. I'm not yet sure if we need `SignupActions` or could use something simpler but from the first looks it seems like yes, we need it.

The solution might be to use `page` for redirecting to checkout as you do if we are on `/plans` page and provide `cartItem` if not. What do you think?